Objects/Converters/ConverterRevit/ConverterRevitShared/Partial Classes/ConvertColumn.cs

```diff
@@ -197,8 +197,7 @@ public RevitColumn ColumnToSpeckle(DB.FamilyInstance revitColumn)
       //make line from point and height
       if (baseLine == null && baseGeometry is Point basePoint)
       {
-        var topLevelParam = revitColumn.get_Parameter(BuiltInParameter.FAMILY_TOP_LEVEL_PARAM);
-        var elevation = (double)((RevitLevel)ParameterToSpeckle(topLevelParam).value).elevation;
+        var elevation = ConvertAndCacheLevel(revitColumn, BuiltInParameter.FAMILY_TOP_LEVEL_PARAM).elevation;
```

Comment:
Will this throw if the top level is null? Or is it never null?

Reply:
I don't think this can be null. unlike walls, there doesn't seem to be an "unconnected height" concept for cols and 
I didn't manage to make a beam without a top level

Reply:
the issue here before wasn't that there wasn't a top level, it was that the top level param was of type `ElementId` which wasn't handled by `ParameterToSpeckle` (not an issue anymore since we're no longer using this)

Reply:
Gotcha, txs!